utils/argv_fuzzing/argv-fuzz-inl.h

```diff
@@ -63,7 +64,12 @@ static char **afl_init_argv(int *argc) {
   char *ptr = in_buf;
   int   rc = 0;
 
-  if (read(0, in_buf, MAX_CMDLINE_LEN - 2) < 0) {}
+  ssize_t num = read(0, in_buf, MAX_CMDLINE_LEN - 2);
+  if (num < 0) {
+      abort();
+  }
+  in_buf[num] = '\0';
```

Comment:
this is missing a `if (num == 0) ...` returning argc of 0.

Reply:
I didnt't think so. With empty input, we have `in_buf = "\0\0"`. The while condition is false, hence you already get argc = 0

Reply:
Yeah, agree with @afflux